src/python_testing/execute_python_tests.py

```diff
@@ -55,65 +55,52 @@ def main(search_directory, env_file):
+        "TC_MCORE_FS_1_1.py", # This test requires a TH_SERVER application, hence not ready run with CI
+        "TC_MCORE_FS_1_2.py", # This test requires a TH_SERVER application, hence not ready run with CI
+        "TC_MCORE_FS_1_3.py", # This test requires a TH_SERVER application, hence not ready run with CI
+        "TC_MCORE_FS_1_4.py", # This test requires a TH_SERVER application, hence not ready run with CI
+        "TC_MCORE_FS_1_5.py", # This test requires a TH_SERVER application, hence not ready run with CI
+        "TC_OCC_3_1.py", # There are CI issues for the test cases that implements manually controlling sensor device for the occupancy state ON/OFF change
+        "TC_OCC_3_2.py", # There are CI issues for the test cases that implements manually controlling sensor device for the occupancy state ON/OFF change
+        "TC_BRBINFO_4_1.py", # This test requires a TH_ICD_SERVER application, hence not ready run with CI
+        "TestCommissioningTimeSync.py", # Code/Test not being used or not shared code for any other tests
+        "TestConformanceSupport.py", # Unit test - does not run against an app
+        "TestChoiceConformanceSupport.py", # Unit test - does not run against an app
+        "TC_DEMTestBase.py", # Shared code for TC_DEM, not a standalone test
+        "TestConformanceTest.py",  # Unit test of the conformance test (TC_DeviceConformance) - does not run against an app
+        "TestIdChecks.py", # Unit test - does not run against an app
+        "TestSpecParsingDeviceType.py", # Unit test - does not run against an app
+        "TestConformanceTest.py", # Unit test - does not run against an app
+        "TestMatterTestingSupport.py", # Unit test - does not run against an app
+        "TestSpecParsingSupport.py", # Unit test - does not run against an app
+        "TestTimeSyncTrustedTimeSource.py", # Unit test and shared code for scripts/tests/TestTimeSyncTrustedTimeSourceRunner.py
+        "drlk_2_x_common.py", # Shared code for TC_DRLK, not a standalone test
+        "execute_python_tests.py", # Python code that runs all the python tests from src/python_testing (This code itself run via tests.yaml)
+        "hello_external_runner.py", # Code/Test not being used or not shared code for any other tests
+        "hello_test.py", # Is a template for tests
+        "test_plan_support.py", # Shared code for TC_*, not a standalone test
+        "test_plan_table_generator.py" # Code/Test not being used or not shared code for any other tests
     }
 
-    """
-    Explanation for excluded files:
-    The above list of files are excluded from the tests as they are either not app-specific tests
-    or are run through a different block of tests mentioned in tests.yaml.
-    This is to ensure that only relevant test scripts are executed, avoiding redundancy and ensuring
-    the correctness of the test suite.
-    """
+    # Explanation for excluded files:
```

Comment:
do we still need the explanation? I would either add it on top of the variable (to comment on the variable) or delete completely as the list has now its own comments for each element.

Reply:
I think deleting completely would avoid any confusion and avoid duplicate explanation too.